frontend/src/tests/session-tests.js

```diff
@@ -92,7 +94,7 @@ export function sessionsTests(api) {
         expect(updatedSession).toContainObject(newData);
     });
 
```

Comment:
General question for error checking, should we also check the response code?

Reply:
since we only have 200 or 500. I think it's enough to just check status.